packages/graph/hash_graph/lib/graph/src/datastore/postgres/database_type.rs

```diff
@@ -3,9 +3,14 @@ use crate::types::{
     VersionedUri,
 };
 
+/// A type, which is stored in a [`PostgresDatabase`].
+///
+/// [`PostgresDatabase`]: crate::datastore::PostgresDatabase
 pub trait DatabaseType {
+    /// Returns the identifying ID of this type.
```

Comment:
```suggestion
    /// Returns the versioned identifier for this instance of a type.
```

Perhaps something like that?

or

```suggestion
    /// Returns the unique versioned URI used to identify this instance of a type.
```